YoutubeDownloader/ViewModels/Components/DownloadViewModel.cs

```diff
@@ -35,21 +37,28 @@ public class DownloadViewModel : PropertyChangedBase, IDisposable
 
     public CancellationToken CancellationToken => _cancellationTokenSource.Token;
 
+    [AlsoNotifyFor(nameof(IsRunning))]
```

Comment:
I forgot, do we still use `Fody.PropertyChanged` for this?

Reply:
yes, it is still used

Reply:
Is it having trouble resolving notification dependencies here? Does it not work without `AlsoNotifyFor`?

Reply:
The IsRunning is a helper property that I introduced to make bindings easier